.gitignore

```diff
@@ -0,0 +1 @@
+build_artefacts
```

Comment:
Yeah i'm not dogmatic about - just might be a bit confusing! Probably better to just get it in to try it out for now?